### DISCUSSION TITLE
Render Pass issue

### DISCUSSION THREAD

**Discussion by AlexTier** [carson-katri/dream-textures#283] [Q&A]
I made all according the instructions (and also try to make some actions to fix it), but the render stopping on 32-36% at the moment when the add-on start to work.
The simple scene with plane and cube.
I can't figure out how to solve this issue :/

**Comment by carson-katri** [comment]
Have you tried using the CPU for cycles instead of GPU so stable diffusion can utilize more of it? Also, how much VRAM does your system have.

**Reply by AlexTier** [reply]
Thanks for reply. Yes, I tried both CPU and GPU. I have 3080Ti, 32 Gb RAM, Intel Core i9-10850K. The RAM load on render is about 15-20%.
If need more info - I can provide it.
Thanks.

**Reply by carson-katri** [reply]
FYI I’m referring specifically to VRAM usage.

**Reply by carson-katri** [reply]
Also what size did you specify in output properties? It will use this for the generation size

**Reply by AlexTier** [reply]
1. VRAM - 12GB
2. 512x512

**Comment by jeffmorice** [comment]
I was having the same problem. In my case, blender crashed after denoising was complete and before processing with Dream. I also tried CPU and GPU. 

I was putting together an example scene to attach to this post ([example blend file](https://drive.google.com/file/d/1oN6r0NCXb4Aw9gXoyd19a6C5kmb4Q3rm/view?usp=sharing)) when it finally worked for me. I had not yet optimized the Tile Size (Properties > Render Properties tab > Performance > Memory > Tile Size) for my GPU. It was set to the default, 2048. When I set it to 256, Blender crashed. When I set it to 512, the render succeeded.

Try setting your tile size to a number equal to or greater than your render resolution.

**Reply by carson-katri** [reply]
My understanding was that you do not need to use tiling in Cycles X unless you are rendering very large images. i have not tested the render pass when tiling, so it probably makes some assumptions RE progressive rendering. SD does not do well with large images anyways, so perhaps catching this and aborting early would be a good workaround.

**Reply by jeffmorice** [reply]
That will definitely be an improvement if it gives a descriptive error message.

As for Cycles X, I am certainly not up on the latest news. My optimization practice may be out of date.

@AlexTier, please let us know if this suggestion fixes your issue.

**Reply by AlexTier** [reply]
@jeffmorice , I set different values for Tile Size, turned off, but the result is the same. I also tried to change other render settings, but it not works

**Reply by jeffmorice** [reply]
Hi @AlexTier. Would you mind sharing your render resolution (ex: 512 x 512) and your Tile Size (ex: 512 or greater)? You will want to make sure your Tile Size is greater than both numbers. For example, if your resolution was 1024 x 512, you would want to set your tile size to 1024 or greater.

I have one more suggestion. This helped me when I was attempting to use Dream with an older, pre-existing file. Try this. Go to Properties > View Layer Properties tab > Passes > Data and ensure the Z pass is unchecked. Otherwise Dream pass will not take place and render will come out transparent.

I hope that works. That's all I have from my experience of this addon so far. If not, then your issue is probably different from the ones I experienced.

**Comment by jclevet** [comment]
Dream Textures work perfectly in the Image Editor. But always freezing at "Starting..." point in render mode.
I tried CPU/GPU, Supported/Experimetal, Multijitter/Sobol, 1 thread/Autodetect, Tiling 256/512/1024, no Z, no Combine... same result.
(Blender3.3.1, DrmTxtr0.0.8, RTX2060, Win10)

**Reply by carson-katri** [reply]
Can you open "Window > Toggle System Console" and then render and see if anything is logged?

**Reply by jclevet** [reply]
It's the right reflex.
There seem to be issues with addons.
I started from a blank Blender and it works!
I will put back my addons one by one.
Thank you for your help and your work.
